app/jobs/modsulator_job.rb

```diff
@@ -35,25 +35,23 @@ def perform(apo_id, uploaded_filename, output_directory, user_login, filetype, x
       if response_xml.nil?
         log.puts('argo.bulk_metadata.bulk_log_error_exception Got no response from server')
         log.puts("argo.bulk_metadata.bulk_log_job_complete #{Time.now.strftime(TIME_FORMAT)}")
+        return
```

Comment:
Non-local exit from iterator, without return value. `next`, `break`, `Array#find`, `Array#any?`, etc. is preferred.